src/lib/components/Modal/README.mdx

```diff
@@ -478,15 +435,38 @@ There are two ways to position buttons within the ModalFooter:
               primaryButtonRef={modalPrimaryButtonRef}
             >
               <ModalHeader>
-                <ModalTitle>Delete the user?</ModalTitle>
+                <ModalTitle>Footer justification</ModalTitle>
                 <ModalCloseButton onClick={() => setModalOpen(false)} />
               </ModalHeader>
               <ModalBody>
                 <ModalContent>
-                  <p>
-                    Do you really want to delete the user <code>admin</code>?
-                    This cannot be undone.
-                  </p>
+                  <Radio
+                    label="Footer justification"
+                    onChange={(e) => setModalJustify(e.target.value)}
+                    options={[
+                      {
+                        label: 'Start',
```

Comment:
I would consider adding to the  labels the actual value passed to `justify` prop of the `<ModalFooter>` component. Like this as a user I have to view source what to use.

Reply:
<img width="719" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/220687404-6430cb0c-26d4-4c0a-8819-ff7097014b29.png">